src/StoriesApi.php

```diff
@@ -46,7 +46,10 @@ public function all(?StoriesRequest $request = null): StoriesResponse
         $request ??= new StoriesRequest();
 
         $response = $this->client->request('GET', self::ENDPOINT, [
-            'query' => $request->toArray(),
+            'query' => [
+                $request->toArray(),
+                'version' => null !== $request->version ? $request->version->value : $this->version->value,
```

Comment:
```suggestion
                'version' => null !== $request->version ? $request->version : $this->version->value,
```

we already have the value in toArray(), right?